execution_workflows/QAPA/README.md

```diff
@@ -0,0 +1,11 @@
+# APAeval nextflow pilot run
+
+### Steps to run this:
+ - To run this first download the required reference files from [here](https://drive.google.com/drive/folders/1MUMilzaqef9u0sjScxzgi0JPTKAFQq_T?usp=sharing)
+ - replace "path_to" in samplesheet_example_files.csv with the path to the reference files downloaded from the Google Drive link
+ - check the path to this directory with `pwd` and replace the `[pwd]` in samplesheet_example_files.csv with the path from the `pwd` command
+ - Before running `QAPA`, you have to install `QAPA`, and the guide to install is [here](https://github.com/morrislab/qapa). Make sure that QAPA is on your $PATH
```

Comment:
So QAPA itself is not inside a container?

Reply:
Oops, I haven't change the `README`. `QAPA` is included in a [docker container](https://hub.docker.com/r/yuukiiwa/qapa/tags?page=1&ordering=last_updated)